src/control/XournalMain.cpp

```diff
@@ -266,16 +266,19 @@ auto XournalMain::run(int argc, char* argv[]) -> int {
     gchar** optFilename = nullptr;
     gchar* pdfFilename = nullptr;
     gchar* imgFilename = nullptr;
+    gboolean* showVersion = nullptr;
```

Comment:
This sis a bug and must be  
```suggestion
    gboolean showVersion = false;
```

Reply:
Thanks for the feedback. I'll change this tomorrow.

Reply:
Done this already, because it was a small change and I wanted to merge your work, but then I saw, that your commits aren't linked to your GitHub account. So I decided to ask you first, if you care.

Reply:
Ah I see. I don't care to much. If you want to go ahead and merge it it's fine with me.